apps/api/src/app/workflows-v2/workflow.controller.e2e.ts

```diff
@@ -199,8 +203,31 @@ describe('Workflow Controller E2E API Testing', () => {
           }
         }
       });
+
+      it('should show digest control value issues when illegal value provided', async () => {
+        const steps = [{ ...buildDigestStep({ controlValues: { amount: '555', unit: 'days' } }) }];
+        const { issues, status } = await createWorkflowAndReturnStepIssues({ steps }, 0);
+        expect(status).to.equal(WorkflowStatusEnum.ERROR);
+        expect(issues.controls).to.deep.equal({
+          amount: [
+            {
+              issueType: 'TIER_LIMIT_EXCEEDED',
+              message:
+                'The maximum delay allowed is 90 days.Please contact our support team to discuss extending this limit for your use case.',
+            },
+          ],
+          unit: [
+            {
+              issueType: 'TIER_LIMIT_EXCEEDED',
+              message:
+                'The maximum delay allowed is 90 days.Please contact our support team to discuss extending this limit for your use case.',
+            },
+          ],
+        });
+      });
     });
```

Comment:
I think the msgs are too specific, every basic change to the script will cause tests to fail, maybe just validate the amount.issue_type and unit.issue_type

Reply:
sure, we can test issueType instead of all of the object.